RevenueCatUI/Data/Localization.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  Localization.swift
+//
+//
+//  Created by Nacho Soto on 7/20/23.
+//
+
+import Foundation
+import RevenueCat
+
+enum Localization {
+
+    /// - Returns: an appropriately short abbreviation for the given `unit`.
+    static func abbreviatedUnitLocalizedString(
+        for unit: NSCalendar.Unit,
+        locale: Locale = .current
+    ) -> String {
+        let (full, abbreviated) = self.unitLocalizedString(for: unit, locale: locale)
+
+        if full.count <= Self.unitAbbreviationMaximumLength {
+            return full
+        } else {
+            return abbreviated
+        }
+    }
+
+    static func localizedDuration(
```

Comment:
This was added in a previous PR, but I moved it here so all localization stuff is together.